expyfun/visual/_visual.py

```diff
@@ -826,6 +826,76 @@ def __init__(self, ec, colors=('w', 'k')):
         self.set_radius(1, 1, units='pix')
 
 
+class ProgressBar(object):
+    """A progress bar that can be displayed between sections
+
+    This uses two rectangles, one outline, and one solid to show how much
+    progress has been made in the experiment.
+
+    Parameters
+    ----------
+    ec : instance of ExperimentController
+        Parent EC.
+    pos : array-like
+        4-element array-like with X, Y center and width, height.
+    units : str
+        Units to use. These will apply to all spatial aspects of the drawing.
+        Must be either ''norm'' or ''pix''.
```

Comment:
I think this should be: `backtick backtick singlequote norm singlequote backtick backtick or backtick backtick singlequote pix singlequote backtick backtick`

Reply:
@drammock you can save yourself some pain in this instance by using the triple-backtick code block:
```
``'norm'`` or ``'pix'``
```